src/feedpropertiesdialog.cpp

```diff
@@ -19,624 +19,678 @@
 #include "feedpropertiesdialog.h"
 
 FeedPropertiesDialog::FeedPropertiesDialog(bool isFeed, QWidget *parent)
-  : Dialog(parent)
-  , isFeed_(isFeed)
+    : Dialog(parent)
```

Comment:
These whitespace changes are not related to the merge request, are they? It would be nicer and cleaner to not have them then.

Reply:
These whitespace are made by the original developer not me.

Reply:
GitHub claims those whitespace changes have been done by you. Maybe your editor has auto-formatting enabled?

Reply:
No.

Reply:
I do not understand. There is a diff (by you) and it clearly states that the whitespace changes. Who should it be if not you who added that whitespace??

Reply:
Oddly, perhaps my Qt Creator added them, but not me.

Reply:
It seems to affect other files too. A diff of >50k lines is not expected for such a comparably small change.

Reply:
I totaly agree with you.